### PR TITLE
Enable supervisorctl utility

### PR DESCRIPTION
**Pull request by aramalipoor**
Users can use https://github.com/reverbdotcom/migr8 to migrate small amounts of data. Wanted to include it in the image but will not do it for now.

### CODE REVIEW

File: 4.0.6/Dockerfile [hunk 1]
```diff
@@ -5,12 +5,12 @@ RUN apk update && \
...
 COPY redis/ /redis/config
-COPY supervisord.conf /
+COPY supervisord.conf /etc/supervisord.conf
 COPY start.sh /start.sh
 
 RUN chmod +x /start.sh && \
-    chgrp -R 0 /var/run /var/log /redis/config && \
-    chmod -R g=u /var/run /var/log /redis/config
+    chgrp -R 0 /var/run /var/log /redis/config /etc/supervisord.conf && \
+    chmod -R g=u /var/run /var/log /redis/config /etc/supervisord.conf
```
Comment (aliscott) [inline 4.0.6/Dockerfile]: Why does `/etc/supervisord.conf` need to be writeable?

**Comment by aliscott** [comment]
Looks good. One question.

File: 4.0.6/Dockerfile (hunk 1 above)
Comment (aramalipoor) [reply 4.0.6/Dockerfile]: If the user needs to run `supervisorctl` it should be readable, but not write TBH :thinking:

Comment (aliscott) [reply 4.0.6/Dockerfile]: OK, this will already be readable. These permissions change it to be group writeable.

Comment (aramalipoor) [reply 4.0.6/Dockerfile]: That was weird for me too, `cat /etc/supervisord.conf` couldn't read the file!

Comment (aliscott) [reply 4.0.6/Dockerfile]: What were the read permissions before and after this? Was it not group readable before?

Comment (aramalipoor) [reply 4.0.6/Dockerfile]: I tested again and you're right it is already group-readable, I don't why it wasn't readable, but now it is, so will remove this ;)

**Pull request merged**